### DISCUSSION TITLE
Define "Security" formally

### DISCUSSION THREAD

**Discussion by kedder** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#108] [Suggestions] [answered]
Most transaction objects reference something, called "security id" (defined as "Identifier for the security which the transaction applies to"). For example: [Stock Issuance Transaction](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/docs/schema/objects/transactions/issuance/schema-objects-transactions-issuance-stock_issuance.md). However, I can't find the formal definition of what this "Security" is or what fields it consists of. I'd find it useful to have some kind of pointer to the definition of "Security" in the descriptions of fields, referencing it.

**Comment by jacobyavis** [comment, marked as answer]
Good point @kedder. This is especially confusing as it applies to an issuance object, e.g. [the Stock Issuance object](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/docs/schema/objects/transactions/issuance/schema-objects-transactions-issuance-stock_issuance.md)
I suggest changing the description of "security_id"

Instead of simply:

"Identifier for the security which the transaction applies to"

We can bolster this up to better describe the intended usage of this field as so:

"Identifier for the security (stock, plan security, warrant, or convertible) by which it can be referenced by other transaction objects. Note that while this identifier is created with an issuance object, it should be different than the issuance object's `id` field which identifies the issuance transaction object itself. All future transactions on the security (e.g. acceptance, transfer, cancel, etc.) must reference this `security_id` to qualify which security the transaction applies to."

**Reply by kedder** [reply]
Hm, IIUC, there's an "issuance _transaction_ id", and "stock security id", which are different. Do I understand correctly that the term "Security" here refers to something so abstract that it has no specific properties other than the "id"? Is something other than stock considered "security" as well (e.g. convertible)?

**Reply by jacobyavis** [reply]
Yes there are four different types of securities in the model, so we use "security" as the generic term to mean one of a "stock", a "convertible", a "warrant", or a "plan security" (e.g. stock option or RSU, or anything issued from a plan)

**Reply by kedder** [reply]
Thanks @jacobyavis, this makes sense. I think adding this explanation to the description of the field (or having it in some article with basic concepts) would definitely help to wrap a head around it.

**Reply by JSv4** [reply]
See Issue #126
